setup.py

```diff
@@ -26,7 +26,7 @@ def load_about():
 
 setup(
     name="tutor-indigo",
-    version=ABOUT["__version__"],
+    version=ABOUT["__package_version__"],
```

Comment:
This is not going to work, because `__package_version__` is no longer defined in `__about__.py`.

Reply:
Yes, good catch.